xbmc/pvr/windows/GUIWindowPVRBase.cpp

```diff
@@ -85,6 +85,13 @@ std::string CGUIWindowPVRBase::GetSelectedItemPath(bool bRadio)
   return m_selectedItemPaths[bRadio];
 }
 
+void CGUIWindowPVRBase::ResetObservers(void)
```

Comment:
looks like an orphan method or do you call it anywhere else?
btw my comments are lost after your force-push.